desc/compute/_deprecated.py

```diff
@@ -0,0 +1,382 @@
+"""Deprecated compute functions.
+
+These are kept for verification purposes. They do not
+appear in the public documentation under the list of variables.
+"""
+
+from functools import partial
+
+from orthax.legendre import leggauss
+
+from desc.backend import imap, jit, jnp
+
+from ..integrals.bounce_integral import Bounce1D
+from ..integrals.quad_utils import (
+    automorphism_sin,
+    chebgauss2,
+    get_quadrature,
+    grad_automorphism_sin,
+)
+from ..utils import cross, dot, safediv
+from ._fast_ion import _cvdrift0, _drift1, _drift2, _v_tau
+from ._neoclassical import _bounce_doc, _dH, _dI
+from .data_index import register_compute_fun
+
+_bounce1D_doc = {
+    "num_well": _bounce_doc["num_well"],
+    "num_quad": _bounce_doc["num_quad"],
+    "num_pitch": _bounce_doc["num_pitch"],
+    "quad": _bounce_doc["quad"],
+}
+
+
+def _compute(fun, fun_data, data, grid, num_pitch, simp=False, reduce=True):
```

Comment:
Does the AD chain go through each iteration of the loop when you do multiple alpha or rho values?

Reply:
How would I test this